Source/WTF/wtf/threads/Signals.h

```diff
@@ -56,8 +59,16 @@ enum class Signal {
     AccessFault, // For posix this is both SIGSEGV and SIGBUS
     NumberOfSignals = AccessFault + 2, // AccessFault is really two signals.
     Unknown = NumberOfSignals
+#else
+    FloatingPoint,
+    IllegalInstruction,
+    AccessFault,
+    NumberOfSignals = AccessFault + 1,
+    Unknown
```

Comment:
We should make `Unknown = NumberOfSignals` since NumberOfSignals is not a signal.